src/main/cpp/Robot.cpp

```diff
@@ -53,7 +59,18 @@ void Robot::AutonomousPeriodic() {
     m_autonChooser.ResumeAutonomous();
 }
 
-void Robot::TeleopPeriodic() { SubsystemBase::RunAllTeleopPeriodic(); }
+void Robot::TeleopPeriodic() {
+    SubsystemBase::RunAllTeleopPeriodic();
+
+    static frc::Joystick appendageStick1{HWConfig::appendageStick1PortID};
+    static frc::Joystick joyStick1{HWConfig::joyStick1PortID};
```

Comment:
Drive stick? Appendage stick? (Not saying to remove it, a rename would help though)